Sketch.java

```diff
@@ -8,28 +8,54 @@
    */
   public void settings() {
 	// put your size call here
-    size(400, 400);
+    size(500, 500);
   }
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(24, 161, 219);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
 	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+	// Windows
+    rect(130, 290, 50, 50);
+    fill(255, 251, 0);
 
-    stroke(255);
-    line(50, 125, 70, 50);  
+    rect(200, 290, 50, 50);
+    fill(255, 251, 0);
+
+    rect(270, 290, 50, 50);
+    fill(255, 251, 0);
+
+  // Sun
+    ellipse(460, 50, 150, 150);
+    fill(38, 0, 255);
+
+  // House
+    rect(120, 240, 220, 220);
+    fill(0,0,0);
+
+  // Grass
+    rect(0, 450, 500, 70);
+    fill(0, 156, 23);
+
+  // Pathway
+    rect(200,440,70,80);
+    fill(255, 0, 0);
+
+  // Roof
+    triangle(80, 240, 250, 130, 380, 250);
+    fill(0, 0, 0);
```

Comment:
Align your comments with the code they refer to.  They should not be outdented because it make the code less readable